src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Avx.cs

```diff
@@ -143,6 +143,162 @@ public abstract class Avx : Sse42
+        /// The above native signature does not exist. We provide this additional overload for completeness.
+        /// </summary>
+        public static Vector256<float> CompareNotLessThan(Vector256<float> left, Vector256<float> right) => Compare(left, right, FloatComparisonMode.UnorderedNotLessThanSignaling);
+        /// <summary>
+        /// __m256d _mm256_cmpnlt_pd (__m256d a,  __m256d b)
+        ///   CMPPD ymm, ymm/m256, imm8(5)
+        /// The above native signature does not exist. We provide this additional overload for completeness.
+        /// </summary>
+        public static Vector256<double> CompareNotLessThan(Vector256<double> left, Vector256<double> right) => Compare(left, right, FloatComparisonMode.UnorderedNotLessThanSignaling);
+
+        /// <summary>
+        /// __m256 _mm256_cmpnle_ps (__m256 a,  __m256 b)
+        ///   CMPPS ymm, ymm/m256, imm8(6)
+        /// The above native signature does not exist. We provide this additional overload for completeness.
+        /// </summary>
+        public static Vector256<float> CompareNotLessThanOrEqual(Vector256<float> left, Vector256<float> right) => Compare(left, right, FloatComparisonMode.UnorderedNotLessThanOrEqualSignaling);
+        /// <summary>
+        /// __m256d _mm256_cmpnle_pd (__m256d a,  __m256d b)
+        ///   CMPPD ymm, ymm/m256, imm8(6)
+        /// The above native signature does not exist. We provide this additional overload for completeness.
+        /// </summary>
+        public static Vector256<double> CompareNotLessThanOrEqual(Vector256<double> left, Vector256<double> right) => Compare(left, right, FloatComparisonMode.UnorderedNotLessThanOrEqualSignaling);
+
+        /// <summary>
+        /// __m256 _mm256_cmpord_ps (__m256 a,  __m256 b)
+        ///   CMPPS ymm, ymm/m256, imm8(7)
+        /// The above native signature does not exist. We provide this additional overload for completeness.
+        /// </summary>
+        public static Vector256<float> CompareOrdered(Vector256<float> left, Vector256<float> right) => Compare(left, right, FloatComparisonMode.OrderedNonSignaling);
+        /// <summary>
+        /// __m256d _mm256_cmpord_pd (__m256d a,  __m256d b)
+        ///   CMPPD ymm, ymm/m256, imm8(7)
+        /// The above native signature does not exist. We provide this additional overload for completeness.
+        /// </summary>
+        public static Vector256<double> CompareOrdered(Vector256<double> left, Vector256<double> right) => Compare(left, right, FloatComparisonMode.OrderedNonSignaling);
+
+        /// <summary>
+        /// __m256 _mm256_cmpunord_ps (__m256 a,  __m256 b)
+        ///   CMPPS ymm, ymm/m256, imm8(3)
+        /// The above native signature does not exist. We provide this additional overload for completeness.
+        /// </summary>
+        public static Vector256<float> CompareUnordered(Vector256<float> left, Vector256<float> right) => Compare(left, right, FloatComparisonMode.UnorderedNonSignaling);
```

Comment:
I think we keep these in alphabetical order, is that right? If so, these should go after the `CompareScalar` overloads. I see that you've done that in System.Runtime.Intrinsics.cs

Reply:
> I think we keep these in alphabetical order, is that right?

Right, will fix.

> I see that you've done that in System.Runtime.Intrinsics.cs

That's because that one is auto-generated rather than copy/paste 😄